spec/unit/workload_mapper_spec.rb

```diff
@@ -212,6 +212,15 @@
         }
       }
     }
+
+    let(:unlinked_services_one_missing) {
+      {
+       "web" => {
+          "links" => ["db"]
+        }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.